setup.cfg

```diff
@@ -1,6 +1,7 @@
 [flake8]
 max-line-length=100
 exclude = docs,.venv
+ignore = E203, E501, W503, F811
```

Comment:
How do you feel about bumping `max-line-length` to 88 instead of ignoring `E501`? Check this section for details https://github.com/psf/black#line-length

Reply:
The max is actually already set higher than 88, so could probably get rid of that error.

Reply:
I'd like to bring the line length down to 88, in order to maximize the number of side-by-side editor windows that can fit on a screen.

(Personally I was happy with 79, but I'm willing to give 88 a shot since it's the default.)

Reply:
Black is using the default 88, Just need to update the max-line-length argument for flake8